setup.cfg

```diff
@@ -18,14 +18,29 @@ version = 0.6
 [options]
 python_requires = >=3.5
 install_requires =
-    numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'
+    # NOTE: platform_system 'AIX' and platform_machine 'aarch64' are mutually
+    #       exclusive so they are specified by themself when setting their
+    #       numpy version.
+
+    # platform_system=='AIX' requires numpy 1.16.0 (needed for Python < 3.8)
     numpy==1.16.0; python_version=='3.5' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
-    numpy==1.17.3; python_version=='3.8'
+
+    # platform_machine=='aarch64' requires numpy 1.19.0 (needed for Python < 3.9)
+    numpy==1.19.0; python_version=='3.5' and platform_machine=='aarch64'
```

Comment:
This is incorrect, numpy 1.19 doesn't support Python 3.5 anymore. Can you change this to `1.18.5` (no wheel, but it does work)?

Reply:
I didn't think about them dropping Python 3.5. Should it just not be declared?

I know this library doesn't want to pull dependencies that previously existed so it doesn't yank support from downstream dependencies, but what is the proper stance in this case? Should it say that aarch64 requires Python 3.6? I can change it to `1.18.5` as suggested, but I don't know if it works either. I just know wheels were built in 1.19.

Reply:
Poking around a little, and I don't think I have much time to really get to the bottom of it, but it looks like aarch could have been around for awhile: https://github.com/numpy/numpy/pull/11016

More recent request to build wheels:
- https://github.com/numpy/numpy/pull/15279
- https://github.com/numpy/numpy/issues/14886

I'm not sure if the goal of this library is to provide the versions needed to use wheels or if versions building from source are OK. If it's building from source `1.19.2` isn't required for Python 3.6-3.8, but if it's to build from a wheel `1.19.2` is required (just applying this same comment to the other versions).

Reply:
I'm fairly sure `aarch64` in 1.18.5 worked, so I'd add that for completeness. In practice it won't matter much, py35 is ancient history.

Reply:
Ok, I've updated the config. Do you know if it should be earlier than 1.18.5? I'm not sure if that's the oldest version or if doesn't matter as much if it's built from source.

Reply:
I think 1.18.5 is right. There were definitely some useful fixes in the 1.18.x branch, and requiring the most recent bugfix release for a minor version seems like the right thing to do.